salsah/src/public/js/jquery.htmleditor.js

```diff
@@ -165,8 +170,19 @@
 							
 						}
 					},
-					toolbar: [ ['Bold', 'Italic', 'Underline', 'Strike', 'Subscript', 'Superscript', '-','RemoveFormat', 'Link', 'Unlink'] ], // configuration for toolbar buttons, must conform to `STANDARD_MAPPING`
-					removeButtons: ''
+                    extraPlugins: 'codemirror,codesnippet',
+                    toolbarGroups: [
```

Comment:
it explains how the toolbar configuration works, the link might be helpful to understand the config